topics/statistics/metadata.yaml

```diff
@@ -42,6 +42,33 @@ material:
     contributors:
       - marziacremona
       - fabio-cumbo
+      
+  -
+    title: "Basics of machine learning"
+    type: "tutorial"
+    name: "machinelearning"
+    galaxy_tour: no
+    hands_on: no
+    slides: yes
+    workflows: yes
+    questions:
+      - "What is machine learning?"
+      - "Why is it useful?"
+      - "What are its different approaches?"
+    objectives:
+      - "Provide the basics of machine learning and its variants."
+      - "Learn how to make predictions using the training and test data."
+      - "Classify data using a Galaxy workflow."
+    requirements:
+    time_estimation: "1/2h"
+    key_points:
+      - "Machine learning algorithms learn features from data."
+      - "It is used for multiple tasks like classification, regression, clustering and so on."
+      - "For the classification task, data is divided into training and test parts."
+      - "Each data sample in training and test sets has a category/class."
+      - "Many learning tasks can be performed on datasets using Galaxy tools for machine learning."
+    contributors:
+      - Anup Kumar
```

Comment:
Can you use your GitHub username here and add yourself to the CONTRIBUTORS.yaml file? Thanks :smile: